.circleci/config.yml

```diff
@@ -40,6 +41,25 @@ jobs:
             - ~/.gradle
           key: v1-dependencies-{{ checksum "build.gradle" }}
 
+      - run:
+          name: Acceptance tests
+          working_directory: ~/repo/http_server_spec
+          command: |
+            sudo apt update
+            sudo apt install autoconf bison build-essential libssl-dev libyaml-dev libreadline6-dev zlib1g-dev libncurses5-dev libffi-dev libgdbm6 libgdbm-dev
+            git clone https://github.com/rbenv/rbenv.git ~/.rbenv
+            echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc
+            echo 'eval "$(rbenv init -)"' >> ~/.bashrc
+            source ~/.bashrc
+            git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build
+            PATH="$HOME/.rbenv/bin:$PATH"
+            rbenv install 2.5.0
+            rbenv global 2.5.0
+            sudo apt-get install rubygems ruby-dev
+            sudo gem install bundler:1.17.2
+            bundle install
+            rake test:f1
```

Comment:
FWIW usually you can reference a shell script instead of doing and inline script. I like to use an inline script when there are only a couple of steps, but then move to a dedicated shell script once the task(s) is more involved. YMMV.

Reply:
I wasn't sure how to go about building a ruby environment.